test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -1242,13 +1242,22 @@ public void TensorFlowImageClassification()
             IDataView trainDataset = trainTestData.TrainSet;
             IDataView testDataset = trainTestData.TestSet;
 
-            var pipeline = mlContext.Model.ImageClassification(
-                "ImagePath", "Label",
-                arch: ImageClassificationEstimator.Architecture.ResnetV2101,
-                epoch: 5,
-                batchSize: 5,
-                learningRate: 0.01f,
-                testOnTrainSet: false);
+            var validationSet = mlContext.Transforms.LoadImages("Image", fullImagesetFolderPath, true, "ImagePath") // true indicates we want the image as a VBuffer<byte>
+                    .Fit(testDataset)
+                    .Transform(testDataset);
+
+            var pipeline = mlContext.Transforms.LoadImages("Image", fullImagesetFolderPath, true, "ImagePath") // true indicates we want the image as a VBuffer<byte>
+                .Append(mlContext.Model.ImageClassification(
+                    "Image", "Label",
+                    // Just by changing/selecting InceptionV3 here instead of 
+                    // ResnetV2101 you can try a different architecture/pre-trained 
+                    // model. 
+                    arch: ImageClassificationEstimator.Architecture.ResnetV2101,
+                    epoch: 50,
```

Comment:
Please make sure to sync to master as Aishwarya has also modified this piece of code in her PR.